pyproject.toml

```diff
@@ -174,9 +174,11 @@ show_missing = true
 # https://mypy.readthedocs.io/en/stable/config_file.html
 [tool.mypy]
 # Import discovery
+packages = ["torchgeo", "tests"]
+exclude = ["^tests/data"]
```

Comment:
We were inadvertently ignoring these in the previous `exclude` because we forgot a `^`. As a result, these files are quite far from passing type checks. Let's continue to ignore them for now.